scripts/Diffraction/isis_powder/osiris.py

```diff
@@ -141,6 +269,12 @@ def _generate_out_file_paths(self, run_details, unit=""):
         out_file_names["output_name"] = os.path.splitext(os.path.basename(out_file_names["nxs_filename"]))[0]
         return out_file_names
 
+    def should_subtract_empty_inst(self):
```

Comment:
Sorry - I see this now. It's used in focus.py. I still wonder if it can be removed though...

I think this needed for the scenario where `subtract_empty_can=False`. In this case the code would attempt to subtract an empty instrument if you didn't have this function. For `subtract_empty_can=True` the empty instrument would be ignored and an empty can subtraction would be done instead. So all fine